v4/source/introduction.yaml

```diff
@@ -214,23 +214,183 @@ tags:
 
       The list of Mattermost WebSocket events are:
       - added_to_team
+        ```
+        {
+          event: 'added_to_team',
+          data: {
+            team_id: '<team_id>',
+            user_id: '<user_id>'
+          },
+          broadcast: {
+            omit_users: null,
+            user_id: '<user_id>',
+            channel_id: '<channel_id>',
+            team_id: '<team_id>'
+          },
+          seq: '<seq_id>'
+        }
+        ```
       - authentication_challenge
-      - channel_converted
+        ```
+        {
+          token: <token-id>
+        }
+        ```
       - channel_created
+        ```
+        {
+          'broadcast': {
+            'omit_users': None,
+            'channel_id': '<channel_id>',
+            'team_id': '<team_id>',
+            'user_id': '<user_id>'
+          },
+          'data': {
+            'channel_id': '<channel_id>',
+            'team_id': '<team_id>'
+          },
+          'event': 'channel_created',
+          'seq': 9
+        }
+        ```
       - channel_deleted
-      - channel_member_updated
+        ```
+        {
+          'event': 'channel_deleted',
+          'data': {
+            'channel_id': '<channel_id>'
+           },
+           'broadcast': {
```

Comment:
The ordering of the fields `event`, `data`, and `broadcast` is different across the events. To make it easier to read, could we go with a consistent ordering?